src/index.ts

```diff
@@ -5,10 +5,20 @@ import { JsonRPCRequest, JsonRPCResponse } from "web3/providers";
 
 const { fetch, Headers } = fetchPonyfill();
 
+const RATE_LIMIT_STATUS = 429;
+const DEFAULT_MAX_RETRIES = 3;
+const DEFAULT_RETRY_INTERVAL = 1000;
+const DEFAULT_RETRY_JITTER = 250;
```

Comment:
One thing I want to make sure I get review on is whether these defaults make sense. The time between retries is the interval plus a random number up to the jitter, in milliseconds. Is a one second interval too long? Anything less than that may just run into another rate limit, since we cap on queries per second.

Reply:
I think these sound fine for now